paddle/phi/kernels/set_inplace_kernel.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2024 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
`inplace` is a concept in the level of `ops`(in the `ops.yaml`), representing the space where ops's output can write to the space of input. so in the level of `kernel`, there is no concept of `inplace`, because both `inplace ops` and `non-inplace ops` use  the same computational logic and the same kernel.
so, we suggest filename is `set_kernel.h`

Reply:
revised